src/groupeddataframe/splitapplycombine.jl

```diff
@@ -1090,10 +1027,32 @@ function _agg2idx_map_helper(idx, idx_agg)
 end
 
 function _combine(f::AbstractVector{<:Pair},
-                  gd::GroupedDataFrame, nms::AbstractVector{Symbol})
+                  gd::GroupedDataFrame, nms::AbstractVector{Symbol},
+                  copycols::Bool, keeprows::Bool) # TODO: remove these defaults
```

Comment:
What defaults?

Reply:
sorry - I have forgotten to remove the comment